rts/lra-examples/cdi-embedded/src/main/resources/META-INF/beans.xml

```diff
@@ -2,5 +2,5 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee 
 		http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
-       bean-discovery-mode="annotated">
+       bean-discovery-mode="all">
```

Comment:
@marcosgopen are you able to share why this is needed, please?

Reply:
It was a test, I now removed that